### ISSUE TITLE
All-day events are displayed in front of the sticky header

### ISSUE THREAD

**Issue by maradragan** [fullcalendar/fullcalendar#5596]
In v5, I have a calendar for which I have set height='auto'.
When in the timeGridWeek or timeGridDay view, if I have several all-day events and scroll, the all-day events overlap the sticky days header:
see https://codepen.io/maradragan/pen/oNbdvZZ

As a workaround I've done the following
```
.fc .fc-timegrid .fc-daygrid-body {
  z-index: 1;
}
```
but I'm not sure it doesn't break other things.

**Comment by gjvoosten** [comment]
The code says this: https://github.com/fullcalendar/fullcalendar/blob/master/packages/timegrid/src/styles/timegrid.css#L5
Either z-index 2 is too high, or the header elements, that have a z-index of 2 now, need a z-index of 3: https://github.com/fullcalendar/fullcalendar/blob/master/packages/common/src/styles/scrollgrid.css#L79

**Comment by acerix** [comment]
Thanks, confimed.

**Comment by arshaw** [comment]
the root problem is that `fc-daygrid-body` has a z-index of 2 which is beating out `.fc .fc-scrollgrid-section-sticky>*`, which also has z-index of 2 but is earlier in the DOM, so it's losing

**Comment by gjvoosten** [comment]
@arshaw But now the _more_ popup goes behind the sticky days header. So you traded one problem for another…

**Comment by arshaw** [comment]
thanks for pointing this out @gjvoosten . i will fix that before officially releasing

**Comment by arshaw** [comment]
I just reverted the changes I made because they cause new bugs as @gjvoosten points out. This problem is too hairy to fix easily before the next release, so I need to punt on it for now. @gjvoosten you mention you'd like to have the popover be a direct child of the root calendar (.fc) div as it was in v4. I've come to the same conclusion that this is the best solution. It will also fix some other bugs as well (#5532)
